public/js/angular/controllers.js

```diff
@@ -615,13 +624,24 @@ angular
           .attr("src", config.gogglesUrl + "/en-US/webxray.js")
           .attr("data-baseuri", config.gogglesUrl + "/en-US")[0]
         );
+        analytics.event('Goggles activated');
       });
 
       // toggle additional help text
       $('.goggles-help-button').click(function () {
+        analytics.event('Goggles help button clicked');
```

Comment:
AFAICS, there's not actually a way to close the help text (lol). Adding a label won't tell us anything, I think.